src/DynamoCoreWpf/Views/GuidedTour/GuideBackground.xaml.cs

```diff
@@ -0,0 +1,68 @@
+﻿using Dynamo.Wpf.ViewModels.GuidedTour;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using System.Windows;
+using System.Windows.Controls;
+using System.Windows.Data;
+using System.Windows.Documents;
+using System.Windows.Input;
+using System.Windows.Media;
+using System.Windows.Media.Imaging;
+using System.Windows.Navigation;
+using System.Windows.Shapes;
+
+namespace Dynamo.Wpf.Views.GuidedTour
+{
+    /// <summary>
+    /// Interaction logic for GuideBackground.xaml
+    /// </summary>
+    public partial class GuideBackground : UserControl
+    {
+        public GuideBackground()
+        {
+            InitializeComponent();
+            DataContext = this;
```

Comment:
@Astul-Betizagasti  I'm not sure if is a good idea of having the Model and ViewModel inside the View (GuideBackground), thinking that in the future the HoleRect  will be dynamic (that could be a Rect or UIElement) and probably will be read (deserialized) from a json file. I guess we should create a GuideBackgroundViewModel class and define all the properties used inside that ViewModel, please let me know what you think. Thanks

Reply:
It was my understanding that the common practice for UserControls is not to have a viewmodel (I could be wrong here). Although I do agree that it could be a viable option. @QilongTang do you have any preferences?